src/components/Dialogs/Multistepper/Multistepper.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import Dialog from '../GeneralDialog/GeneralDialog'
+import { DialogProps } from '@/components/Dialogs/GeneralDialog/GeneralDialog'
+import {
+  StyledHeader,
+  StyledStepsInfoContainer,
+  StyledStepInfo,
+  StyledCircle,
+  StyledStepInfoText,
+  StyledExitButton,
+} from './Multistepper.style'
+import { Icon } from '@/shared/components'
+
+type Step = {
+  title: string
+  element: JSX.Element
+}
+
+type MultistepperProps = {
+  steps: Step[]
+  currentStep?: number
```

Comment:
`currentStep` suggests the prop is of `Step` type - let's use something else, for example `currentStepIdx`